modmail/extensions/extension_manager.py

```diff
@@ -198,7 +198,9 @@ async def rewalk_extensions(self, ctx: Context) -> None:
         log.debug(f"Refreshing list of {self.type}s.")
 
         # make sure the new walk contains all currently loaded extensions, so they can be unloaded
-        loaded_extensions = dict((en, sl) for en, sl in self.all_extensions.items() if en in self.bot.extensions)
+        loaded_extensions = dict(
+            (en, sl) for en, sl in self.all_extensions.items() if en in self.bot.extensions
+        )
```

Comment:
en -> ext_name, sl -> should_load

best to break this into a natural for loop, honestly.